src/components/VolunteerForm/RequestForHelpForm.js

```diff
@@ -60,6 +60,19 @@ const RequestForHelpForm = (props) => {
           />
         </Form.Item>
 
+        <Form.Item
```

Comment:
There is an NOP field defined with validations in Fields/input.js, please use that.

Reply:
I have used nop field from Input.js file as you mentioned. But made small change in input.js. export const NOPField = () => (
  <Form.Item
    label="Number of Persons"
    name="nop"
    rules={[
      {
        type: "number",
        transform: toNumber,
        message: "Invalid format",
      },
      {
        required: true,
        message: "Number of Persons is required",
      },
    ]}
  >
    <Input
      type="number"
      min="1"
      maxLength={5}
      placeholder="Enter number of individuals who need help"
    />

Adding {min:1} dint work in rules object, it was saving with 0 as value for NOP. So removed that number validation from rules and added min=1 and type="number" in input markup. Because min=1 in markup was again not working without type="number". Now it throws error for value 0 also does not allow user to enter character.. Hope it is fine?

Reply:
yes, looks good